server/src/internalClusterTest/java/org/elasticsearch/indices/memory/breaker/HierarchyCircuitBreakerTelemetryIT.java

```diff
@@ -41,54 +39,11 @@
 import static org.elasticsearch.indices.breaker.HierarchyCircuitBreakerService.TOTAL_CIRCUIT_BREAKER_LIMIT_SETTING;
 
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0, numClientNodes = 0, supportsDedicatedMasters = true)
-public class HierarchyCircuitBreakerTelemetryTests extends ESIntegTestCase {
+public class HierarchyCircuitBreakerTelemetryIT extends ESIntegTestCase {
 
     @Override
     protected Collection<Class<? extends Plugin>> nodePlugins() {
-        return List.of(TestCircuitBreakerTelemetryPlugin.class);
-    }
-
-    public static class TestCircuitBreakerTelemetryPlugin extends TestTelemetryPlugin {
```

Comment:
Cleanup: we don't need to subclass `TestTelemetryPlugin`, it's possible (and better) to use it directly